src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardViewModel.swift

```diff
@@ -77,8 +77,26 @@ class HomeStatisticsCardViewModel {
 		} else {
 			primaryTitle = String(format: AppStrings.Statistics.Card.Incidence.date, "")
 		}
-
 		primarySubtitle = AppStrings.Statistics.AddCard.localCardPrimarySubtitle
+		
+		if let sevenDayHospitalizationTrend = regionStatisticsData.sevenDayHospitalizationIncidence, let sevenDayHospitalizationIncidenceUpdatedAt = regionStatisticsData.sevenDayHospitalizationIncidenceUpdatedAt {
+			secondaryValue = sevenDayHospitalizationTrend.formattedValue
+			secondaryTrendImage = sevenDayHospitalizationTrend.trendImage
+			secondaryTrendImageTintColor = sevenDayHospitalizationTrend.trendTintColor
+			secondaryTrendAccessibilityLabel = sevenDayHospitalizationTrend.trendAccessibilityLabel
+			secondaryTrendAccessibilityValue = sevenDayHospitalizationTrend.trendAccessibilityValue
+
+			let sevenDayHospitalizationIncidenceUpdatedDate = Date(timeIntervalSince1970: TimeInterval(sevenDayHospitalizationIncidenceUpdatedAt))
+			secondaryTitle = sevenDayHospitalizationIncidenceUpdatedDate.formatted(
+				todayString: AppStrings.Statistics.Card.Combined7DaysIncidence.today,
+				yesterdayString: AppStrings.Statistics.Card.Combined7DaysIncidence.yesterday,
+				otherDateString: AppStrings.Statistics.Card.Combined7DaysIncidence.date
+			)
+		} else {
+			secondaryTitle = String(format: AppStrings.Statistics.Card.Incidence.date, "")
+		}
+		secondarySubtitle = AppStrings.Statistics.Card.Combined7DaysIncidence.secondaryLabelSubtitle
```

Comment:
This one must differentiate between local statistics for a federal state or administrative unit and for an administrative unit it should say "Hospitalisierungen in {federal state name}" (See Figma)